tests/components/uk_transport/test_sensor.py

```diff
@@ -80,18 +86,21 @@ def test_train(self, mock_req):
         with requests_mock.Mocker() as mock_req:
             uri = re.compile(UkTransportSensor.TRANSPORT_API_URL_BASE + "*")
             mock_req.get(uri, text=load_fixture("uk_transport_train.json"))
-            assert setup_component(self.hass, "sensor", {"sensor": self.config})
+            assert setup_component(self.hass, "sensor",
+                                   {"sensor": self.config})
 
         train_state = self.hass.states.get("sensor.next_train_to_WAT")
 
         assert type(train_state.state) == str
-        assert train_state.name == "Next train to {}".format(TRAIN_DESTINATION_NAME)
-        assert train_state.attributes.get(ATTR_STATION_CODE) == TRAIN_STATION_CODE
-        assert train_state.attributes.get(ATTR_CALLING_AT) == TRAIN_DESTINATION_NAME
+        assert train_state.name == "Next train to {}".format(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)